p2-layouts-master/code/style.css

```diff
@@ -0,0 +1,209 @@
+/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+
+body {
+  background-color: rgb(255, 248, 213);
+}
+
+header {
+  background-image: url('watermelon.JPG');
```

Comment:
Nice with a background image! To make the header go all the way to the edges, you could add `margin: 0;` to the `body`.